awx/main/migrations/0185_hop_nodes.py

```diff
@@ -32,4 +32,15 @@ class Migration(migrations.Migration):
             model_name='instancelink',
             constraint=models.CheckConstraint(check=models.Q(('source', models.F('target')), _negated=True), name='source_and_target_can_not_be_equal'),
         ),
+        migrations.AlterField(
+            model_name='instance',
+            name='listener_port',
+            field=models.PositiveIntegerField(
+                blank=True,
+                default=None,
+                help_text='Port that Receptor will listen for incoming connections on.',
+                null=True,
+                validators=[django.core.validators.MinValueValidator(1), django.core.validators.MaxValueValidator(65535)],
```

Comment:
oh can you append this to the top of the migration file
```python3
import django.core.validators
from django.db import migrations, models
```

we are getting import errors in the api-test CI